src/tooling/piral-cli/src/apps/new-pilet.test.ts

```diff
@@ -26,11 +26,12 @@ jest.mock('../common/clients/npm', () => {
 describe('New Pilet Command', () => {
   it('scaffolding in an empty directory works', async () => {
     jest.setTimeout(60000);
+
     const dir = createTempDir();
     await newPilet(dir, {
       install: false,
       source: 'piral@latest',
-      registry: 'https://someFakeRegistry.com',
+      registry: 'https://registry.npmjs.org',
```

Comment:
This was set to some invalid URL intentionally. Did it stop working with that setting?

Reply:
It seems like an invalid registry is checked earlier in the process and an error is thrown.  This particular test is "scaffolding in an empty directory works" and the error was causing a premature termination of the test.  We could make this a valid registry as above, or remove the line altogether, which will also default to `https://registry.npmjs.org`.

In either case, there is a separate test for a truly invalid registry somewhere around Line #43

Reply:
npm7 may also be the cause of this, where an error is now thrown with the invalid registry, when it did not do so with npm6.  `scripts.ts` may be the proper place to fix this.

Reply:
> In either case, there is a separate test for a truly invalid registry somewhere around Line 43

Agreed we can go for that and remove the registry property from this one.